docs/src/main/tut/docs/core/interpreters/README.md

```diff
@@ -24,12 +24,12 @@ import freestyle._
 import cats.implicits._
 
 @free trait KVStore {
-  def put[A](key: String, value: A): OpSeq[Unit]
-  def get[A](key: String): OpSeq[Option[A]]
-  def delete(key: String): OpSeq[Unit]
+  def put[A](key: String, value: A): FS[Unit]
+  def get[A](key: String): FS[Option[A]]
+  def delete(key: String): FS[Unit]
   def update[A](key: String, f: A => A): OpSeq[Unit] =
-    get[A](key) flatMap {
-      case Some(a) => put[A](key, f(a))
+    get[A](key).freeS flatMap {
```

Comment:
This is the one point in which unifying both types adds a bit of chrome: now we need to manually raise the `FS[A]` to a `FreeS[ $F, A]`. The same happens with the result of `put` in the next line.